src/main/java/seedu/uninurse/logic/commands/EditPatientCommand.java

```diff
@@ -100,12 +101,20 @@ private static Patient createEditedPatient(Patient patientToEdit, EditPatientDes
         Phone updatedPhone = editPatientDescriptor.getPhone().orElse(patientToEdit.getPhone());
         Email updatedEmail = editPatientDescriptor.getEmail().orElse(patientToEdit.getEmail());
         Address updatedAddress = editPatientDescriptor.getAddress().orElse(patientToEdit.getAddress());
-        ConditionList updatedConditions = patientToEdit.getConditions(); // editing of conditions is not supported
-        TaskList updatedTasks = patientToEdit.getTasks(); // editPatient command does not allow editing tasks
+
+        // editing of conditions is not supported
+        ConditionList updatedConditions = patientToEdit.getConditions();
+
+        // editPatient command does not allow editing medications
+        MedicationList updatedMedicationList = patientToEdit.getMedications();
```

Comment:
Perhaps keeping the names consistent with the other list attributes would be better.
```suggestion
        MedicationList updatedMedications = patientToEdit.getMedications();
```